earth_enterprise/src/installer/install_server.sh

```diff
@@ -629,6 +748,81 @@ check_server_running()
   fi
 }
 
+# Candidate to move to common
+is_valid_alphanumeric()
+{
+  # Standard function that tests a string to see if it passes the "valid" alphanumeric for user name and group name.
+  # For simplicity -- we limit to letters, numbers and underscores.
+  regex="^[a-zA-Z_]{1}[a-zA-Z0-9_-]*$"
+
+  if [ ! -z "$1" ] && [[ $1 =~ $regex ]]; then
+    return 0
+  else
+    return 1
+  fi
+}
+
+# Show config values and prompt user for confirmation to continue install
+prompt_install_confirmation()
+{
+  local backupStringValue=""
+
+  if [ $BACKUPSERVER == true ]; then
+    backupStringValue="YES"
+  else
+    backupStringValue="NO"
+  fi
+
+  echo -e "\nYou have chosen to install $GEES with the following settings:\n"
+  echo -e "# CPU's: \t\t\t$NUM_CPUS"
+  echo -e "Operating System: \t\t$MACHINE_OS_FRIENDLY"
+  echo -e "64 bit OS: \t\t\tYES"	# Will always be true because we exit if 32 bit OS
+  if [ $IS_NEWINSTALL == false ]; then
+    echo -e "Backup Server: \t\t\t$backupStringValue"
+  fi
+  echo -e "Publisher Root: \t\t${PUBLISHER_ROOT_VOLUME_INFO[0]}"
+  echo -e "Publisher Root Mount Point: \t${PUBLISHER_ROOT_VOLUME_INFO[3]}"
+  echo -e "Install Location: \t\t$BASEINSTALLDIR_OPT"
+  echo -e "Postgres User: \t\t\t$GEPGUSER_NAME"
+  echo -e "Apache User: \t\t\t$GEAPACHEUSER_NAME"
+  echo -e "Group: \t\t\t\t$GRPNAME"
+  echo -e "Disk Space:\n"
+
+#	GRPNAME="gegroup"
+#	GEPGUSER_NAME="gepguser"
+
+  # display disk space
+  df -h | grep -v -E "^none"
+
+  echo ""
+
+  if ! prompt_to_quit "X (Exit) the installer and cancel the installation - C (Continue) to install/upgrade."; then
+    return 1
+  else
+        echo -e "\nProceeding with installation..."
+    return 0
+    fi
+}
+
+load_systemrc_config()
+{
+  if [ -f "$SYSTEMRC" ]; then
+    # read existing config information
+    ASSET_ROOT=$(xml_file_get_xpath "$SYSTEMRC" "//Systemrc/assetroot/text()")
+    GEFUSIONUSER_NAME=$(xml_file_get_xpath "$SYSTEMRC" "//Systemrc/fusionUsername/text()")
+    GRPNAME=$(xml_file_get_xpath "$SYSTEMRC" "//Systemrc/userGroupname/text()")
+  fi
+}
+
+# Get user/group info from previous installation.  Uses user/group for particular files/directories
+get_server_installation_info()
+{
+  GEAPACHEUSER_NAME=$(get_default_user "$PUBLISHER_ROOT/search_space" $GEAPACHEUSER_NAME)
+  GRPNAME=$(get_default_group "$PUBLISHER_ROOT/search_space" $GRPNAME)
+  GEPGUSER_NAME=$(get_default_user "$PGSQL_DATA" $GEPGUSER_NAME)
+  GEFUSIONUSER_NAME=$(get_default_user "$ASSET_ROOT" $GEFUSIONUSER_NAME)
```

Comment:
Fusion might not be installed on the same machine as server.  So the server install shouldn't need this.